custom_hostname.go

```diff
@@ -61,25 +61,23 @@ type CustomHostnameSSLCertificates struct {
 
 // CustomHostnameSSL represents the SSL section in a given custom hostname.
 type CustomHostnameSSL struct {
-	ID                   string                              `json:"id,omitempty"`
-	Status               string                              `json:"status,omitempty"`
-	Method               string                              `json:"method,omitempty"`
-	Type                 string                              `json:"type,omitempty"`
-	CnameTarget          string                              `json:"cname_target,omitempty"`
-	CnameName            string                              `json:"cname,omitempty"`
-	TxtName              string                              `json:"txt_name,omitempty"`
-	TxtValue             string                              `json:"txt_value,omitempty"`
-	Wildcard             *bool                               `json:"wildcard,omitempty"`
-	CustomCertificate    string                              `json:"custom_certificate,omitempty"`
-	CustomKey            string                              `json:"custom_key,omitempty"`
-	CertificateAuthority string                              `json:"certificate_authority,omitempty"`
-	Issuer               string                              `json:"issuer,omitempty"`
-	SerialNumber         string                              `json:"serial_number,omitempty"`
-	Settings             CustomHostnameSSLSettings           `json:"settings,omitempty"`
-	ValidationErrors     []CustomHostnameSSLValidationErrors `json:"validation_errors,omitempty"`
-	HTTPUrl              string                              `json:"http_url,omitempty"`
-	HTTPBody             string                              `json:"http_body,omitempty"`
-	Certificates         []CustomHostnameSSLCertificates     `json:"certificates,omitempty"`
+	ID                   string                          `json:"id,omitempty"`
+	Status               string                          `json:"status,omitempty"`
+	Method               string                          `json:"method,omitempty"`
+	Type                 string                          `json:"type,omitempty"`
+	Wildcard             *bool                           `json:"wildcard,omitempty"`
+	CustomCertificate    string                          `json:"custom_certificate,omitempty"`
+	CustomKey            string                          `json:"custom_key,omitempty"`
+	CertificateAuthority string                          `json:"certificate_authority,omitempty"`
+	Issuer               string                          `json:"issuer,omitempty"`
+	SerialNumber         string                          `json:"serial_number,omitempty"`
+	Settings             CustomHostnameSSLSettings       `json:"settings,omitempty"`
+	Certificates         []CustomHostnameSSLCertificates `json:"certificates,omitempty"`
+	// Deprecated, use ValidationRecords.
```

Comment:
just to make sure this works in editors properly

```suggestion
	// Deprecated: use ValidationRecords.
```